features/events/syslog/src/main/resources/org/opennms/netmgt/syslogd/grok-patterns.txt

```diff
@@ -1,6 +1,13 @@
 # Note: Ordering is important in this file. Patterns should be ordered from most specific to least specific. The first
 # pattern found to match will be the one used.
 
+# Patterns for syslog messages parsed incorrectly during OCE audit
```

Comment:
Can we just say we're adding additional support for Cisco syslog messages, rather than referring to the "audit"?